flopy/discretization/grid.py

```diff
@@ -603,6 +607,22 @@ def cross_section_set_contour_arrays(
     def map_polygons(self):
         raise NotImplementedError("must define map_polygons in child class")
 
+    def get_lni(self, *nodes):
+        """
+        Get the layer index and within-layer node index (both 0-based).
+        if no nodes are specified, all are returned in ascending order.
+
+        Parameters
+        ----------
+        nodes : the node numbers (zero or more ints)
+
+        Returns
+        -------
+            A tuple (layer index, node index), or a
+            list of such if multiple nodes provided
+        """
+        return get_lni([self.ncpl for _ in range(self.nlay)], *nodes)
```

Comment:
It seems like:
```python
        ncpl = (
            [self.ncpl for _ in range(self.nlay)]
            if isinstance(self.ncpl, int)
            else list(self.ncpl)
        )

        return get_lni(ncpl, *nodes)
```
could go here, in `Grid` and that would remove the need for having a seperate `get_lni()` call in  the `UnstructuredGrid` class.

Reply:
Thanks- updated the `Grid` method to reflect this and removed override in `UnstructuredGrid`